src/test/kotlin/com/mparticle/kits/GoogleAnalyticsFirebaseGA4KitTest.kt

```diff
@@ -1058,9 +1058,33 @@ class GoogleAnalyticsFirebaseGA4KitTest {
     @Test
     fun testScreenNameSanitized() {
         kitInstance.logScreen("Some long Screen name", null)
+        val firebaseScreenViewEvent = firebaseSdk.loggedEvents[0]
         TestCase.assertEquals(
             "Some_long_Screen_name",
-            FirebaseAnalytics.getInstance(null)?.currentScreenName
+            firebaseScreenViewEvent.value.getString("screen_name")
+        )
+    }
+
+    @Test
+    fun testScreenNameAttributes() {
+        val attributes = hashMapOf<String, String>()
+        attributes["testAttributeKey"] = "testAttributeValue"
+        kitInstance.logScreen("testScreenName", attributes)
+        val firebaseScreenViewEvent = firebaseSdk.loggedEvents[0]
+        // even though we are passing one attribute, it should contain two including the screen_name
+        TestCase.assertEquals(
+                2,
```

Comment:
Same formatting issue here, use this by the following option.
Toolbar -> Code -> Reformat File